spec/models/form526_submission_spec.rb

```diff
@@ -1282,7 +1282,17 @@ def expect_no_max_cfi_logged(diagnostic_code)
       context 'with submission confirmation email when successful job statuses' do
         subject { create(:form526_submission, :with_multiple_succesful_jobs) }
 
-        it 'returns one job triggered' do
+        it 'does not trigger job when disability_526_call_received_email_from_polling enabled' do
+          allow(Flipper).to receive(:enabled?).with(:disability_526_call_received_email_from_polling,
```

Comment:
hey Mark, the tests are now failing because someone added a Flipper called `saved_claim_schema_validation_disable` which is called in this call chain, i'm guessing when the form526 submission record is created on line 1283, which requires a saved_claim.

all we have to do is add this line above where we're calling the Flipper mock.
```
allow(Flipper).to receive(:enabled?).with(:saved_claim_schema_validation_disable).and_return(false)
```
i think that should work